### PR TITLE
Panels bug fix2

### PR DESCRIPTION
**Pull request by ps48**
### Description
Part 2 of Operational Panel bugs.

### Issues Resolved
1. Fixed import bug in add visualization flyout. 
2. Added refresh panel action
3. Remove Flyout Preview Button
4. Change maxItems in homepage
5. Fixed Styling and margin for visualizations
6. Fixed Add new visualization - edge case bug

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx [hunk 1]
```diff
@@ -46,6 +44,7 @@ import {
   VisualizationType,
 } from '../../../../../common/types/custom_panels';
 import './visualization_flyout.scss';
+import { uiSettingsService } from '../../../../../common/utils';
```
Comment (joshuali925) [inline dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx:47]: unused?

File: dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts [hunk 2]
```diff
@@ -67,6 +74,7 @@ export class CustomPanelsAdaptor {
     try {
       const response = await client.callAsCurrentUser('observability.getObject', {
         objectType: 'operationalPanel',
+        maxItems: 10000,
```
Comment (joshuali925) [inline dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts:77]: will this cause a performance issue? might be better to use pagination in the future

**Review by joshuali925 — APPROVED** [review]

File: dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx (hunk 1 above)
Comment (ps48) [reply dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx:47]: This is a bug fix, the code using the import is already in [main](https://github.com/opensearch-project/trace-analytics/blob/cd516e5108156362a382ddca575fe228d7cf0223/dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx#L208).

File: dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts (hunk 2 above)
Comment (ps48) [reply dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts:77]: Yes, this should be changed in future. For now I took the constant from [reporting](https://github.com/opensearch-project/dashboards-reports/blob/93d885c641c8c7fc8beeb081bee6ffd30b44f73b/dashboards-reports/server/routes/utils/constants.ts#L86-L87).

**Review by mengweieric — APPROVED** [review]

**Pull request merged**